tests/test.py

```diff
@@ -1,4 +1,54 @@
+import sys
+sys.path.append('../')
 from handler import stalk
+import requests
+import os
 
 # Yeah Testing is cool and totally worth it!
-print(stalk('aashutoshrathi'))
+user = 'aashutoshrathi'
+delimiter = '</strong>'
+os.environ['CONTRI_API'] = 'https://github-contributions-api.herokuapp.com/'
```

Comment:
Also, can we have one test for CONTRI_API?

Reply:
Can you explain more on this? I can't figure out a way to test this.

Reply:
We can just check if that's a number of today or not.

Reply:
Done